.github/workflows/ci-linux-shared-local-fuzztest.yml

```diff
@@ -0,0 +1,71 @@
+# Builds libavif with local aom and dav1d and runs fuzztest tests
+# (tests that have 'fuzztest' in their name).
+# They are run in "unit test mode" (run with random inputs for a short period of time)
+# see https://github.com/google/fuzztest/blob/main/doc/quickstart-cmake.md
+# Runs on ubuntu only. libavif is built with clang.
+
+name: CI Unix Shared Local Fuzztest
+on: [push, pull_request]
+
+permissions:
+  contents: read
+
+# Cancel the workflow if a new one is triggered from the same PR, branch, or tag, except on main.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+jobs:
+  build-shared-local:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-24.04]
+        libyuv: [OFF]
+        include:
+          - runs-on: ubuntu-24.04
+            compiler: gcc
```

Comment:
no need to specify this as clang is necessary for fuzztest (cf `CMAKE_C_COMPILER=clang`)